### DISCUSSION TITLE
Burn crashes when using XML from HyperlinkSidebarTheme.xml

### DISCUSSION THREAD

**Discussion by chrpai** [wixtoolset/issues#6889] [Announcements]
* Which version of WiX are you building with?

3.11.2

* Describe the problem and the steps to reproduce it.

Download HyperlinkLargeTheme.xml from GitHub, create a wix bootstrapper using WixStandardBootstrapperApplication.HyperlinkLicense and set the ThemeFile to the file path of the theme.  Build and run the EXE.

* Describe the behavior you expected and how it differed from the actual behavior.

I expected the screen to render.  Instead the application crashes and the log shows:

[44B4:154C][2022-08-29T14:19:54]e000: Error 0x8007000d: Failed to load theme controls.
[44B4:154C][2022-08-29T14:19:54]e000: Error 0x80004005: Failed to create main window.

HyperlinkTheme.xml and HyperlinkLargeTheme.xml both work fine. It's only the HyperlinkSidebarTheme.xml that's giving me trouble. Strange since the commit for all of them is 6 years ago.  If I'm missing something, it's escaping me.

**Comment by rseanhall** [comment]
The theme references an image that is likely missing. You need to include all images in the bundle. It helps to load it in thmviewer.exe to make sure it works properly.

**Reply by chrpai** [reply]
I tried loading in theme viewer. It didn't give me any errors just showed a tree of pages.

**Comment by chrpai** [comment]
This was my fault. I didn't realize I had to change the BA Ref to WixStandardBootstrapperApplication.HyperlinkSidebarLicense.  The page I was looking at only talked about Rtf and Hyperlink.

It'd be really good if this was a build / validation error rather then a runtime error.

**Reply by rseanhall** [reply]
I don't think anyone has created a feature request for that, yet. Today all of the relevant code to handle the theme is written in C++ so it's not a simple request.
